tests/test_iam_rules.py

```diff
@@ -72,5 +73,40 @@ def test_match_resource():
             statement.actions_expanded,
             resource,
             statement.effect,
+            "123456768124"
     )
     assert rule.matches(target_arn) == {"Read"}
+
+
+def test_dont_match_different_account_():
+    """Tests a rule does not match an AWS resource defined in a different
+    account than the rule."""
+    document = """
+        {
+            "Statement": [
+                {
+                    "Action": [
+                        "dynamodb:Scan"
+                    ],
+                    "Effect": "Allow",
+                    "Resource": [
+                        "arn:aws:dynamodb:*:*:table/table*"
+                    ]
+                }
+            ],
+            "Version": "2012-10-17"
+        }
+    """
+    target_arn = "arn:aws:dynamodb:eu-west-1:111111111111:table/table2"
+    document = json.loads(document)
+    policy = Policy(document)
+    statement = policy.statements[0]
+    resource = statement.resources.pop()
+    rule = ARNRule(
+                        statement.action_summary(),
+                        statement.actions_expanded,
+                        resource,
+                        statement.effect,
+                        "123456768124"
+        )
```

Comment:
Fix indentation.

Reply:
Watch out, 8ff40a6 does not fix it. It should be 4 spaces not 8.

Reply:
True thanks, I miss indented it to 8 spaces. Also there was the same indentation problems this same file a few lines above, I'm also fixit it.